Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch

```diff
@@ -85,13 +85,16 @@ index 324afd2a5e50a3b1fd9de926f6da8836b00e173b..865c1e4cf18b25ecf2ba8d592d9ca40f
  import org.bukkit.craftbukkit.block.data.CraftBlockData;
  import org.bukkit.craftbukkit.inventory.CraftItemStack;
  import org.bukkit.craftbukkit.legacy.CraftLegacy;
-@@ -393,6 +394,22 @@ public final class CraftMagicNumbers implements UnsafeValues {
+@@ -393,6 +394,25 @@ public final class CraftMagicNumbers implements UnsafeValues {
              throw new RuntimeException();
          }
      }
 +
 +    @Override
 +    public String getTranslationKey(Material mat) {
++        if (mat.isBlock()) {
++            return getBlock(mat).getDescriptionId();
++        }
```

Comment:
Missing start and end comments.

Reply:
this is code added by paper, there is a start comment many lines above and an end one many lines below

Reply:
ah ok